siac/rentercmd.go

```diff
@@ -271,7 +274,7 @@ func renterfilesloadasciicmd(ascii string) {
 	}
 }
 
-// renterfilesrenamecmd is the handler for the command `siac renter rename [path] [newpath]`.
+// renterfilesrenamecmd is the handler for the command `siac renter mv [path] [newpath]`.
```

Comment:
comment should not talk about the aliases, should talk about the primary. Same for the other 3